### DISCUSSION TITLE
Getting Started - Example Showcase - Missing

### DISCUSSION THREAD

**Discussion by najak3d** [FirstGearGames/FishNet#482] [Feature Request]
I'm totally new here.  Just came over to compare FishNet to Netcode GO.   I was hoping to be able to do a comparison, but as of yet, no luck.

I started out on the deprecated FishNET Examples repo -- which wasn't marked as deprecated -- so wasted an hour there, before I realized it was defunct.  Why wasn't it marked as such?

Then moved on to starting Unity 2023.2 on the FishNET GitHub downloaded.   Only here, no clear "examples" to run.   Just some "Additive Scenes" -- nothing for me to "see in action", for comparison.

So then I moved on to the "FishNet-3rdPersonPrediction" sample -- in Unity 2023 -- ended up with pink materials, and some significant issues with compiler errors (even after installing a half-dozen packages). 

Still, I have no examples to view.   When I run this example, the "Start Server/Client" buttons don't do anything.

So -- I tried this with Unity's "Multiplayer Play Mode 0.4.0" - which works well for NGO, to see if it might work well...   no dice.  The extra render windows have defects not seen in the main editor window.

===
As a noobie coming to "compare", I wasn't expecting to work so hard to see FishNet in Action.

Perhaps it will go smoother by trying an older Unity version, like 2021?  The Github repos do NOT give me any notes on "recommended Unity Version" - but by the age of last check-in, I can "guess" at Unity 2021 LTS, probably. 

===
So boiling this down - PLEASE MAKE THE 1ST TIME USER EXPERIENCE 1000% BETTER.

1. Mark the deprecated Github Examples Repro as "Deprecated!"-- big bold letters.   Tell users where to go.
2. For the examples you direct us to -- they should be more explicit on recommended version of Unity Editor, and dependencies.
3. Would be nice if there was a "showcase" project with MANY scenes.... each demonstrating a different aspect of FishNet.
4. Integrate "ParallelSync" into the Samples -- because this is VITAL.  It's SO MUCH EASIER to be able to turbo-test in the editor without messing with full builds!  And now you are competing against Unity's MPPM, and so it's vital to show this feature in the Showcase examples.

I have money to support you, but was hoping to find something more "turnkey" and "dummy-proof" for the sample projects, which should be maintained, and advertised on front-page everywhere.   I'm going to give this one-more-stab tomorrow, using Unity Editor 2021 -- hopefully with better results.  I'm out of time for now.

**Comment by FirstGearGames** [comment]
These links have been removed. They might be lingering in a couple spots but they have not been updated for awhile.
We're moving to just having things in the Demos folder of FishNet.
